test/Microsoft.AspNetCore.Mvc.Formatters.Json.Test/JsonInputFormatterTest.cs

```diff
@@ -254,7 +254,7 @@ public class JsonInputFormatterTest
         [InlineData("application/some.entity+json;v=2", true)]
         [InlineData("application/some.entity+xml", false)]
         [InlineData("application/some.entity+*", false)]
-        [InlineData("text/some.entity+json", false)]
+        [InlineData("text/some.entity+json", true)]
```

Comment:
This is interesting that it works. JsonInputFormatter only supports `application/*+json` and not `text\*+json` (not sure why this isn't added too)
https://github.com/aspnet/Mvc/blob/release/2.2/src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonInputFormatter.cs#L141

Reply:
I think it would be nice if we have couple of tests verifying with the output formatter selector too (`DefaultOutputFormatterSelectorTest`) @dougbu ?

Reply:
Nah, I think we're fine with the lower-level tests

Reply:
Now that text/json is a superset of text\*+json, this passes. If that isn't the intended behavior, we may need to change the JsonInputFormatter?

Reply:
I actually don't think we need to add any json syntax either: https://github.com/aspnet/Mvc/blob/release/2.2/src/Microsoft.AspNetCore.Mvc.Formatters.Json/JsonInputFormatter.cs#L141